docs/content/howto/notebook.md

```diff
@@ -121,6 +121,33 @@ rrb.Vertical(
   <source media="(max-width: 1200px)" srcset="https://static.rerun.io/notebook_blueprint_example/eb0663a9a8a0de8276390667a774acc1bc86148e/1200w.png">
 </picture>
 
+## Streaming data
+
+The notebook integration supports streaming data to the viewer during cell execution.
+
+You can call `rr.notebook_show()` at any point after calling `rr.init()`, and any
+`rr.log()` calls will be sent to the viewer in real-time.
+
+```python
+import math
+from time import sleep
+
+import numpy as np
+import rerun as rr
+from rerun.utilities import build_color_grid
+
+rr.init("rerun_example_notebook")
+rr.notebook_show()
+
+STEPS = 100
+twists = math.pi * np.sin(np.linspace(0, math.tau, STEPS)) / 4
```

Comment:
```suggestion
twists = math.tau * np.sin(np.linspace(0, math.tau, STEPS)) / 8
```

Reply:
I think we should prefer pi over tau since it's more accessible. (Fewer people know about tau)